### DISCUSSION TITLE
如何安装ssl证书？

### DISCUSSION THREAD

**Discussion by li7355608** [xfgryujk/blivechat#161] [提问] [answered]
如题，在Windows服务器上部署后，如何安装ssl证书将访问协议更改为https

**Comment by xfgryujk** [comment, marked as answer]
搭个反向代理服务器。或者自己改代码加上 `ssl_options` https://www.tornadoweb.org/en/stable/httpserver.html#tornado.httpserver.HTTPServer

**Comment by li7355608** [comment]
好的，已解决！
